src/lib/tools/index.ts

```diff
@@ -0,0 +1,3 @@
+export * from "./fetchUser"
+export * from "./fetchUserArtistFollows"
+export * from "./fetchShowsNearLocation"
```

Comment:
Setting up for easier imports.

I believe this pattern of re-exporting everything from a root file is frowned on in Force due to tree-shaking(?) concerns, so maybe we don't do it there? But seems fine for quantum.

Reply:
Please avoid these types of index exports! Even though this is an experimental app, we don't know where it will finally go. Its impossible to express how much time and effort has gone into basically (all) of our apps unwinding this index pattern; The confusion it generates coupled with VSCode auto import is great, and it makes refactoring opaque. Please trust me 🙏

Reply:
Ha, I was hoping to get your input on this 😄 

Yeah, can definitely back away from this, and I think the imports will also read nicer if they are from a top-level `tools` directory, as suggested above. I def trust you on this, but do you have a handy reference to a discussion about the problems with this approach. I was trying to remember but honestly could not recall the headaches it caused.

Reply:
I wrote a small summary here: https://artsy.slack.com/archives/C2TQ4PT8R/p1671568357903639

TLDR: 100% of the time, refactors become a real headache. And the gain is zero, compared to just concrete single locations that cannot be interpreted. A file import path is like a memory address. 1:1.

Reply:
Thanks for that refresher. Unwound in https://github.com/artsy/quantum/pull/14/commits/fab86c8954b80941e3cb3cfb5ebc1e8737ef1e60